### DISCUSSION TITLE
Nzyme do not monitore same SSID

### DISCUSSION THREAD

**Discussion by pponicki** [nzymedefense/nzyme#539] [Questions] [answered]
Hello, I have four Access points, they all share the same SSID name. Nzyme allows me to monitore only one network from one AP. What can I do to monitore all four AP? At the bottom I put info from logs:

12:22:24.180 [main] ERROR horse.wtf.nzyme.Main - Invalid configuration. Please refer to the example configuration file or documentation.
horse.wtf.nzyme.configuration.InvalidConfigurationException: SSID [Here my SSID] is defined multiple times. You cannot define a network with the same SSID more than once.
at horse.wtf.nzyme.configuration.BaseDot11ConfigurationLoader.validateNetworks(BaseDot11ConfigurationLoader.java:158) ~[nzyme-1.1.1.jar:?]
at horse.wtf.nzyme.configuration.BaseDot11ConfigurationLoader.validate(BaseDot11ConfigurationLoader.java:114) ~[nzyme-1.1.1.jar:?]
at horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader.validate(LeaderConfigurationLoader.java:427) ~[nzyme-1.1.1.jar:?]
at horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader.(LeaderConfigurationLoader.java:79) ~[nzyme-1.1.1.jar:?]
at horse.wtf.nzyme.Main.main(Main.java:83) [nzyme-1.1.1.jar:?]

Can you help?

**Comment by pponicki** [comment]
Any ideas?

**Comment by MoerkyMark** [comment, marked as answer]
Hi, if you have multiple AccesPoints sharing the same SSID you have to do it like this. My problem with this setup is the security which I will post next in this forum.

ssid: MyOnlySSID
    channels: [1,2,3,4,5,6,7,8,9,10,11,12,13,36,100]
    security: [WPA1-PSK-CCMP, WPA2-PSK-CCMP]
    beacon_rate: 40
    bssids: [
      {
        address: "ab:cd:ef:gh:12:34",
        fingerprints: [ XXX1 ]
      }
      {
        address: "ab:cd:ef:gh:56:78",
        fingerprints: [ XXX2 ]
      }
      {
        address: "ab:cd:ef:gh:34:35",
        fingerprints: [ XXX3 ]
      }
    ]

**Reply by pponicki** [reply]
It worked! Thanks!

**Comment by cos-renegade** [comment]
I have done exactly that, but I get a lot of alerts now
UNEXPECTED_BSSID_BEACON
UNEXPECTED_SSID_BEACON
UNEXPECTED_BSSID_PROBERESP

The alerts are from BSSIDs which are in my config ....
Do I have to disable these alerts?``

In the Assets section it is in there like this:

SSID: xxxxxx
Security:
WPA2-PSK-CCMP
Channels:
1, 11, 44, 48
ACCESS POINTS
BSSID	Fingerprints
xx:xx:xx:xx:xx:f1	
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx9f5e, 
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx6a9a, 
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxcce9
xx:xx:xx:xx:xx:e4	
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx9f5e, 
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx6a9a, 
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxcce9

Do I have to "train" nzyme again?
I edited the config file and added these networks after the initial start.

**Comment by cos-renegade** [comment]
oh ok, it seems to be going away, only 1 alert left: UNEXPECTED_SSID_BEACON

so the answer is patience? :)


edit: nope, all 6 alerts back again :(
